CHANGELOG.md

```diff
@@ -23,13 +23,16 @@ The format is based on Keep a Changelog, and this project adheres to Semantic Ve
 - Electra: Updated block publishing beacon APIs to support Electra.
 - "Submitted builder validator registration settings for custom builders" log message moved to debug level.
 - config: Genesis validator root is now hardcoded in params.BeaconConfig()
+- `grpc-gateway-host` is renamed to http-host. The old name can still be used as an alias.
+- `grpc-gateway-port` is renamed to http-port.
+- `grpc-gateway-corsdomain` is renamed to http-cors-domain.
 
 ### Deprecated
-
+- `--disable-grpc-gateway` flag is deprecated due to grpc gateway removal.
 - `--enable-experimental-state` flag is deprecated. This feature is now on by default. Opt-out with `--disable-experimental-state`.
 
 ### Removed
-
+- removed gRPC Gateway and gRPC versions of REST endpoints ( unused endpoints )
```

Comment:
```suggestion
- removed gRPC Gateway
```
I would remove the second part as it is not really true. What we are really removing is the option to use REST for gRPC endpoints. And I am not sure what you mean by "unused" - someone might be still using them.